docs/API.md

```diff
@@ -301,8 +324,222 @@ Rejects a pending invite for the authenticating account
 await api.rejectRealmInvitation('1234567')
 ```
 
+<details>
+<summary>Output</summary>
+
+No output
+
+</details>
+
+---
+
+### removeRealmInvite
+
+(realmId: string, uuid: string) => Promise\<Realm>
```

Comment:
Might be useful to note that for Bedrock the player's XUID should be provided as the UUID